user_all_groups/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,28 @@
+This module extends the base Odoo module, adding a new field on user form view
+named 'Member of all Groups'.
+
+If checked, the user will belong to all the groups (``res.groups``) and if a new module
+is installed, the user(s) will automatically belong to the new created groups.
+
+This feature can be interesting:
+
+- on production, if you have some 'admin' users that want to access all features.
+
+- in a development environment, when testing or developing new modules, to avoid fastidious
+  initial configuration.
+
+.. figure:: ../static/description/view_res_users_form.png
+
+
+**Note**
+
+- In Odoo Core, some groups are exclusive. This is the case for the three base groups
+  "Access Rights / Portal / Public" and the two account groups "Tax display B2B / Tax display B2C".
+  All that groups are not impacted by this module.
```

Comment:
This formulation is unclear (to me at least), I would reword with something like:

> Users marked as "Member of all Groups" are NOT added to these exclusive groups.

Reply:
done. Thanks.